test/unitTests/core/test-issuers.js

```diff
@@ -0,0 +1,102 @@
+import { test } from 'tape-promise/tape';
+
+import { makeMint } from '../../../core/issuers';
+
+test('split and combine', t => {
+  try {
+    const mint = makeMint('fungible');
+    const issuer = mint.getIssuer();
+    const purse = mint.mint(1000);
+    const payments = [];
+    for (let i = 0; i < 100; i += 1) {
+      payments.push(purse.withdraw(1));
+    }
+
+    const combinedPayment = issuer.combine(payments);
+    t.deepEqual(combinedPayment.getBalance(), issuer.makeAmount(100));
+    for (const payment of payments) {
+      t.throws(() => payment.getBalance());
+    }
+
+    const badAmountsArray = Array(2).fill(issuer.makeAmount(10));
```

Comment:
Please start two new tests here. (goodsplit and badsplit) 
 Each test case should test a single failure mode when possible. The split tests don't depend on the input payment being one that was just built with combine.